src/Data/Algorand/MessagePack/Json.hs

```diff
@@ -16,31 +16,33 @@ module Data.Algorand.MessagePack.Json
 
 import qualified Data.Aeson as JS
 import qualified Data.Aeson.Types as JS
-import Data.ByteString (ByteString)
-import Data.ByteString.Base64 (decodeBase64, encodeBase64)
-import Data.Foldable (toList)
 import qualified Data.HashMap.Strict as HM
 import qualified Data.MessagePack as MP
 import qualified Data.Scientific as S
+
+import Data.ByteString (ByteString)
+import Data.ByteString.Base64 (decodeBase64, encodeBase64)
+import Data.Foldable (toList)
 import Data.Text (Text)
 import Data.Text.Encoding (encodeUtf8)
 
 import qualified Data.Algorand.Address as A
-import Data.Algorand.MessagePack (Canonical (Canonical), EitherError (..), MessagePackObject, MessageUnpackObject, fromAlgoObject, toAlgoObject)
 
+import Data.Algorand.MessagePack (Canonical (Canonical), EitherError (..), MessagePackObject, MessageUnpackObject, fromAlgoObject, toAlgoObject)
```

Comment:
Looks too big. Why is it not auto-formatted?

Reply:
hm.. sometimes hls feels bad (and formatter is built-in to hls). I'll check it

Reply:
AFAIK formatting is done by `stylish-haskell`, so you can try to invoke it manually from terminal and see what it says.